### PR TITLE
🐛 Correct hrefs and upgrade to latest

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
   run:
     runs-on: ubuntu-latest
     container:
-      image: ministryofjustice/cloud-platform-tech-docs-publisher:1.1
+      image: ministryofjustice/cloud-platform-tech-docs-publisher:1.6
```
Comment (petergphillips) [inline .github/workflows/check.yml:10]: latest version grabbed from https://github.com/ministryofjustice/cloud-platform-user-guide/releases

File: Gemfile.lock [hunk 2]
```diff
@@ -1,24 +1,23 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (5.2.4.3)
```
Comment (petergphillips) [inline Gemfile.lock:4]: not sure if we need the `Gemfile.lock` locally, but grabbed from https://github.com/ministryofjustice/cloud-platform-user-guide/blob/main/Gemfile.lock anyway

File: source/auth-user-guide.html.md.erb [hunk 3]
```diff
@@ -48,9 +48,9 @@ When designing your service you should consider whether any of your users have a
 
 While most prison staff will already have a NOMIS account some prison staff might not use NOMIS. And some prison staff that have NOMIS accounts do not use the legacy NOMIS system.
 
-To find out how to onboard DPS users see <a href="#onboarding-users/">onboarding users.</a>
+To find out how to onboard DPS users see [onboarding users](#onboarding-users).
 
-To find out how to create NOMIS user accounts in the Auth developer environment see <a href="#testing-your-service-in-auth/">Testing your service</a>
+To find out how to create NOMIS user accounts in the Auth developer environment see [Testing your service](#getting-access-to-auth-and-testing-your-service)
```
Comment (petergphillips) [inline source/auth-user-guide.html.md.erb:53]: this url was incorrect so fixed

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Pull request merged**